### ISSUE TITLE
a11y bug - episode list - the line before the timestamp should be a CSS border as this is presentation

### ISSUE THREAD

**Issue by jroebu14** [BBC-archive/psammead#4034]
**Describe the bug**
The line before the timestamp should be a CSS border as this is presentation. Using a character to achieve this is bad practice and also this looks odd/is not very clear/readable when there is no CSS 

e.g.
```html
<span aria-hidden=“true” class=“css-mro3c9 e1c0i89j0”>|</span>
```

**To Reproduce**
Steps to reproduce the behaviour:

**Expected behaviour**

**Screenshots**

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Testing notes**

- [ ] This bug fix is expected to need manual testing.

**Additional context**
CC @greenc05
